app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for char in phrase.lower():
+        if char.lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
Rewrite this function to one line with using method `count` of `str`

Reply:
Fixed!